docs/user/api/integrity.md

```diff
@@ -0,0 +1,129 @@
+# Integrity API
+
+<!--[[ preview('user-api-docs') ]]-->
+
+The Integrity API provides access to PyPI's implementation of [PEP 740].
+
+## Concepts
+
+The concepts and objects in the Integrity API closely mirror [PEP 740]:
+
+* **Attestation objects** encapsulate a single "attestation" for a single file,
+  such as a [publish attestation] or [SLSA Provenance].
+
+* **Provenance objects** encapsulate *one or more* attestations for a given
+  file, bundling them with the *identity* that produced them.
+
+The Integrity API deals in provenance objects; users should extract and verify
+individual attestations from a file's provenance, as appropriate.
+
+## Routes
+
+### `GET /integrity/<project>/<version>/<filename>/provenance`
```

Comment:
This renders a little weird in the table of contents due to the length, we might just want to name this route here and provide the URL structure below the heading instead.

![image](https://github.com/user-attachments/assets/9a79d0fd-eddd-4709-8703-3630d7ca3219)